cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const username = 'Qwerty';
+  const password = 'Qwerty123@';
+  const bookAdded = 'Book added to your collection.';
+  const bookDelete = 'Book deleted.';
+  const bookTitle = 'Speaking JavaScript';
+  const bookDescription = 'Like it or not, JavaScript is everywhere these';
 
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/login');
   });
 
-  it('', () => {
+  it('should provide an ability to Log In', () => {
+    cy.get('#userName').type(username);
+    cy.get('#password').type(password);
+    cy.get('#login').click();
+    cy.get('#userName-value').should('contain', username);
+    cy.url().should('contain', '/profile');
+  });
+
+  it('should provide ability to add the book to the collection', () => {
+    cy.login(username, password);
```

Comment:
The command cy.login() is supposed to log in with API but it seems to be used without any implementation of this functionality.